LM-Ontology/ontologies/SACM.sadl

```diff
@@ -6,186 +6,186 @@ import "http://arcos.rack/PROV-S".
 
 /* Base Classes. */
 
-ELEMENT is a type of ENTITY.
-        uuid describes ELEMENT with a single value of type string.
-
-MULTI_LANG_STRING
-    is a type of ELEMENT.
-        hasValue describes MULTI_LANG_STRING with values of type LANG_STRING.
-
-LANG_STRING
-    is a type of ELEMENT.
-        lang describes LANG_STRING with values of type string.
-        lang of LANG_STRING has at most 1 value.
-        contents describes LANG_STRING with values of type string.
-        contents of LANG_STRING has at most 1 value.
-
-EXPRESSION_LANG_STRING
-    is a type of LANG_STRING.
-        expresses describes EXPRESSION_LANG_STRING with values of type EXPRESSION_ELEMENT.
-
-SACM_ELEMENT
-    is a type of ELEMENT.
-        gid describes SACM_ELEMENT with values of type string.
-        gid of SACM_ELEMENT has at most 1 value.
-        isCitation describes SACM_ELEMENT with values of type boolean.
-        isCitation of SACM_ELEMENT has at most 1 value.
-        isAbstract describes SACM_ELEMENT with values of type boolean.
-        isAbstract of SACM_ELEMENT has at most 1 value.
-        cites describes SACM_ELEMENT with values of type SACM_ELEMENT.
-        implements describes SACM_ELEMENT with values of type SACM_ELEMENT.
-
-UTILITY_ELEMENT
-    is a type of SACM_ELEMENT.
-        hasContent describes UTILITY_ELEMENT with values of type MULTI_LANG_STRING.
-
-TAGGED_VALUE
-    is a type of UTILITY_ELEMENT.
-        hasKey describes TAGGED_VALUE with values of type MULTI_LANG_STRING.
-
-NOTE
-    is a type of UTILITY_ELEMENT.
-
-IMPLEMENTATION_CONSTRAINT
-    is a type of UTILITY_ELEMENT.
-
-DESCRIPTION
-    is a type of UTILITY_ELEMENT.
-
-MODEL_ELEMENT
-    is a type of SACM_ELEMENT.
-        hasTaggedValue describes MODEL_ELEMENT with values of type TAGGED_VALUE.
-        hasImplementationConstraint describes MODEL_ELEMENT with values of type IMPLEMENTATION_CONSTRAINT.
-        hasDescription describes MODEL_ELEMENT with values of type DESCRIPTION.
-        hasNote describes MODEL_ELEMENT with values of type NOTE.
-        hasName describes MODEL_ELEMENT with values of type LANG_STRING.
-
-ARTIFACT_ELEMENT
-    is a type of MODEL_ELEMENT.
+Element is a type of ENTITY.
+        uuid describes Element with a single value of type string.
+
+Multi_Lang_String
```

Comment:
I think the Turnstile example would have used `MultiLangString` instead of the underscored version. Is that divergence intended?

Reply:
Another good catch! I'll make the mods.